### ISSUE TITLE
Exposure check performed every time the app is opened

### ISSUE THREAD

**Issue by danielvonmitschke** [corona-warn-app/cwa-app-ios#1522]
## Avoid duplicates

* [x ] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ x] Bug is not already reported in another issue

## Describe the bug

After updating to 1.6.1 on some days exposure checks are performed every time I open the app.
This behavior then suddenly stops and it's back to working the way it's supposed to (exposure checks only once a day).

## Expected behaviour

No additional exposure checks.

## Technical details

- iOS Version: 14.1
- Device: iPhone XS

## Log files

[ExposureChecks-2020-11-17.json.zip](https://github.com/corona-warn-app/cwa-app-ios/files/5551623/ExposureChecks-2020-11-17.json.zip)

## Screenshot

![IMG_0822](https://user-images.githubusercontent.com/1579746/99353736-6d18f880-28a5-11eb-83fe-c4f970e70c4a.PNG)

**Comment by Ein-Tim** [comment]
Hey @danielvonmitschke, This seems like a duplicate of #1517. 
Please take a look at this Issue and consider closing this Issue, it is also intended that the App updates more often than 24h with version 1.6.1.

**Comment by dsarkar** [comment]
Dear @danielvonmitschke, 

Thank you for your contribution. As @Ein-Tim mentions, these two problems have been reported already.  

Please, continue any further contribution regarding duplicate checks here  https://github.com/corona-warn-app/cwa-app-ios/issues/1505. 

And if necessary, continue any further contribution regarding checks performed every time the app is opened here  https://github.com/corona-warn-app/cwa-app-ios/issues/1517.

I will close this issue, so we can focus on the discussions in the existing threads. 

Thanks again! Best wishes,
DS

Corona-Warn-App Open Source Team

**Comment by dsarkar** [comment]
Duplicate of https://github.com/corona-warn-app/cwa-app-ios/issues/1517

**Comment by danielvonmitschke** [comment]
@dsarkar @Ein-Tim 
thanks guys. I looked for a similiar issue but couldn't find it.
